blog/_posts/2025-03-17-enhancing-observability-in-azure-kubernetes-service.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Enhancing Observability in Azure Kubernetes Service (AKS): What's New?"
+description: "Recapping some of the recent announcements and improvements in AKS observability experience."
+date: 2025-03-17
+authors: Pavneet Ahluwalia
+categories: observability
+tags:
+  - azure-monitor
+  - observability
```

Comment:
If observability is a new category, I don't think we should include it as a tag.

```suggestion
```